sdk/core/src/main/java/com/klaviyo/core/config/KlaviyoConfig.kt

```diff
@@ -108,55 +125,52 @@ object KlaviyoConfig : Config {
             if (debounceInterval >= 0) {
                 this.debounceInterval = debounceInterval
             } else {
-                // TODO Logging
+                Registry.log.error(
+                    "${KlaviyoConfig::debounceInterval.name} must be greater or equal to 0"
+                )
             }
         }
 
         override fun networkTimeout(networkTimeout: Int) = apply {
             if (networkTimeout >= 0) {
                 this.networkTimeout = networkTimeout
             } else {
-                // TODO Logging
-            }
-        }
-
-        override fun networkFlushIntervalWifi(networkFlushInterval: Int) = apply {
-            if (networkFlushInterval >= 0) {
-                this.networkFlushIntervals[NetworkMonitor.NetworkType.WIFI.position] = networkFlushInterval
-            } else {
-                // TODO Logging
-            }
-        }
-
-        override fun networkFlushIntervalCell(networkFlushInterval: Int) = apply {
-            if (networkFlushInterval >= 0) {
-                this.networkFlushIntervals[NetworkMonitor.NetworkType.CELL.position] = networkFlushInterval
-            } else {
-                // TODO Logging
+                Registry.log.error(
```

Comment:
There is a separate PR about this, i didn't mean to get this intermingled, but I made a mistake in git. best laid plans...